tests/functional/replication/queueProcessor.js

```diff
@@ -81,6 +81,7 @@ class S3Mock {
                         size: constants.body.length,
                         start: 0,
                         dataStoreName: 'file',
+                        dataStoreETag: `1:${this.contentMd5}`,
```

Comment:
I think I tested this case yesterday. `dataStoreETag` will appear for all objects, mpu or not.